test/functional/config_templates_controller_test.rb

```diff
@@ -79,9 +79,15 @@ class ConfigTemplatesControllerTest < ActionController::TestCase
     ConfigTemplate.find_by_name('PXELinux global default').update_attribute(:template, template)
 
     ProxyAPI::TFTP.any_instance.stubs(:fetch_boot_file).returns(true)
+    ProxyAPI::Template.any_instance.stubs(:template_url).returns('http://proxy.unattended.url:8000')
     Setting[:unattended_url] = "http://foreman.unattended.url"
     @request.env['HTTP_REFERER'] = config_templates_path
 
+    # With Templates Proxy
```

Comment:
The problem I had was there's a TFTP proxy in features, so if it's not in the same test, `ProxyAPI::Template` gets an unexpected invocation for `create_default`.  Any way to avoid that?